web/pgadmin/browser/server_groups/servers/databases/schemas/tables/tests/utils.py

```diff
@@ -593,3 +594,75 @@ def get_table_id(server, db_name, table_name):
     except Exception:
         traceback.print_exc(file=sys.stderr)
         raise
+
+
+def create_postgis_extension(self):
+    """
+    This function will create postgis extension in the database.
+    :return:
+    """
+    try:
+        connection = utils.get_db_connection(self.db_name,
+                                             self.server['username'],
+                                             self.server['db_password'],
+                                             self.server['host'],
+                                             self.server['port'],
+                                             self.server['sslmode'])
+        pg_cursor = connection.cursor()
+        # Check if postgis extension is available to install
+        pg_cursor.execute('''SELECT COUNT(*) FROM pg_available_extensions WHERE
```

Comment:
No need to check if extension exists. The SQL is already doing that "IF NOT EXISTS"

Reply:
The reason for this check is if the extension is not available, then running create extension will raise an exception. "IF NOT EXISTS" checks whether the extension is already created or not.